### PR TITLE
Run deployer as non-root user

### LINKED ISSUE #5769: Deployer POD does not start if SCC restricted "runAsUser" is set to MustRunAsNonRoot
We configured our cluster to run restricted POD's with the MustRunAsNonRoot users. After this change the deployer POD doesn't start. The POD status is Pending. 

We think the problem is that the deployer POD has no USER set. The POD runs with the other modes because it runs as root if the mode is set to RunAsAny or as random user if the mode is set to MustRunAsRange. In MustRunAsNonRoot mode the deployer POD tries to run as root what is not allowed.

### PR DESCRIPTION
**Pull request by ironcladlou**
Fixes #5769

### CODE REVIEW

**Comment by ironcladlou** [comment]
cc @pweil-

File: images/deployer/Dockerfile [hunk 1]
```diff
@@ -9,4 +9,6 @@
 #
 FROM openshift/origin
 
+USER 5555
```
Comment (liggitt) [inline images/deployer/Dockerfile]: warrants a comment, I think

Comment (ironcladlou) [inline images/deployer/Dockerfile]: Added

**Comment by pweil-** [comment]
LGTM

[test]

**Comment by smarterclayton** [comment]
why did you pick 5555?  If it's random, use 1001

**Comment by ironcladlou** [comment]
>why did you pick 5555? If it's random, use 1001

Completely arbitrary. I'll change this and the UID in #5805 to 1001.

**Comment by smarterclayton** [comment]
This is approved if we rebuild, @danmcp due to it being a security issue.

**Comment by smarterclayton** [comment]
It is low risk

**Comment by ironcladlou** [comment]
UID changed to 1001

**Comment by liggitt** [comment]
[test]

**Comment by smarterclayton** [comment]
[merge]

**Comment by sdodson** [comment]
@tdawson if you happen to do the next OSE build we need to make sure that this change makes it into the ose image.

**Pull request merged**